tests/testFormatters.php

```diff
@@ -329,6 +330,77 @@ function testNestedVarExport()
         $this->assertFormattedOutputMatches($expected, 'var_export', $data);
     }
 
+
+    /**
+     * @requires PHP 5.4
```

Comment:
Oooo, shiny. However, I think this should be `@requires PHP 5.5`, as the var dumper component is not available on 5.4. The next `@requires` should work correctly, though, so you could simply omit the php version constraint.

Reply:
VarDUmper 2.8 should work on PHP 5.3. though.

Reply:
In that case you can just get rid of the `--remove symfony/var-dumper`, go back to composer-test-scenarios ^1, and get rid of the `@requires` here. That should clear up the php 5.4 tests.